privacyidea/static/components/token/controllers/tokenDetailController.js

```diff
@@ -1,4 +1,37 @@
 
+function date_object_to_string(date_obj) {
+    s = "";
+    if (date_obj) {
+        Y = date_obj.getFullYear();
+        D = date_obj.getDate();
+        D = (D>9 ? '' : '0') + D;
+        M = date_obj.getMonth()+1;
+        M = (M>9 ? '' : '0') + M;
+        h = date_obj.getHours();
+        h = (h>9 ? '' : '0') + h;
+        m = date_obj.getMinutes();
+        m = (m>9 ? '' : '0') + m;
+        o = date_obj.toTimeString().split(" ")[1];
+        // 10:20:11 GMT+0200 (CEST)
+        o = o.substring(3);
```

Comment:
As you can see i pushed this in a86a0ad.
I rebased my local 644/datepicker branch on master, but was not abel to push it.
Obviously there is no other way then to merge it manually into master. Please drop me a note, when merging is OK.